torch/_higher_order_ops/auto_functionalize.py

```diff
@@ -294,8 +321,10 @@ def sync_update(o, orig_arg):
 
 
 @auto_functionalized.py_functionalize_impl
-def auto_functionalized_func(ctx, _mutable_op, **kwargs):
+def auto_functionalized_func(ctx, _mutable_op, _arg_to_aliased, _all_aliased, **kwargs):
     unwrapped_kwargs = ctx.unwrap_tensors(kwargs)
     with ctx.redispatch_to_next():
-        result = auto_functionalized(_mutable_op, **unwrapped_kwargs)
+        result = auto_functionalized(
+            _mutable_op, _arg_to_aliased, _all_aliased, **unwrapped_kwargs
```

Comment:
ah this is wrong. need to append those to unwrapped_kwargs, but idk if that offend users!